zk/tests/conftest.py

```diff
@@ -87,3 +90,23 @@ def condition():
 
     with docker_run(compose_file, conditions=[condition]):
         yield get_instance
+
+
+@pytest.fixture
+def check():
+    return ZookeeperCheck(CHECK_NAME, {}, [VALID_CONFIG])
+
+
+@pytest.fixture(scope='session')
+def version_metadata():
+    version = os.environ['ZK_VERSION']
+    tokens = version.split('.')
+    major, minor = tokens[0:2]
+    return {
+        'version.scheme': 'semver',
+        'version.major': major,
+        'version.minor': minor,
+        'version.patch': mock.ANY,
+        'version.raw': mock.ANY,
+        'version.release': mock.ANY,
+    }
```

Comment:
The fixture `version_metadata` seems not needed. Since it's defined in `test_metadata` itself.

Reply:
I updated this here: https://github.com/DataDog/integrations-core/pull/4927/commits/717d15a88f36eef7ca4f39b81aee649de0bb5542